github-contribution-guideline.md

```diff
@@ -0,0 +1,122 @@
+# Contribution Guide using Github
+
+This guide describes the process and the conventions for contributing an article using the tools provided by GitHub.
+
+## Prerequisites
+
+1. Get a Github Account if you not yet have one
+   
+2. Make sure you are properly registered as a public member of the "Zühlke Github Community":
+    * Follow these instructions: https://github.com/Zuehlke/core/issues/6
+    * If you can not access this issue: 
+      * send your github username to one of the admins, e.g. Rolf Bruderer
+      * after you received the invitation, accept it
+      * then proceed as described on https://github.com/Zuehlke/core/issues/6
+    
```

Comment:
Being member of "Zühlke Github Community" is not a prerequisite. As I played the process through with a dummy user, I did not need it. 
@bruderol Agree to remove this section?

Reply:
yes, we could probably remove it. But I think it would be a good opportunity to advertise for the github community and get our zühlke members properly registered - also the idea is that only Zühlke members can contribute - isnt it?

Reply:
it certainly is. moved complete explanation to "advanced topics" and referenced it.